validation/chOpenApiValidator_test.go

```diff
@@ -56,9 +58,9 @@ func TestValidateRequestAgainstOpenApiSpecFailsAbs(t *testing.T) {
 // TestValidateRequestAgainstOpenApiSpecFailsFileOpen asserts that when calling to load the file using an ABS path fails
 // it is handled correctly.
 func TestValidateRequestAgainstOpenApiSpecFailsFileOpen(t *testing.T) {
+	callOnce = sync.Once{}
```

Comment:
What does adding a new callOnce every time do for these tests? Is it worth putting a small comment above it just to clarify why we re-create the once? guessing to do with the once == true after first test is run so all others fail.

Reply:
Yes exactly as you said Aaron, in order to test all these unit tests collectively, we need to reset once as this causes the `doc` to not get initialised after subsequent calls